UnityProject/Assets/Scripts/Medical/CloningConsole.cs

```diff
@@ -0,0 +1,192 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+/// <summary>
+/// Main component for cloning console.
+/// </summary>
+public class CloningConsole : MonoBehaviour, IServerSpawn
+{
+	public delegate void ChangeEvent();
+	public static event ChangeEvent changeEvent; // TODO Is this warning true?
```

Comment:
Codacy found an issue: [Remove the unused event 'changeEvent' or invoke it.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6387847)